app/views/_sidebar.html

```diff
@@ -1,58 +1,75 @@
-<nav class="navbar navbar-sidebar">
-    <ul class="nav nav-sidenav-primary">
-      <li ng-if="'service_catalog_landing_page' | enableTechPreviewFeature" class="visible-xs-block">
-        <a href="./"><span class="pficon pficon-home" aria-hidden="true"></span> Home</a>
-      </li>
+<div class="nav-pf-vertical nav-pf-vertical-with-sub-menus"
+  ng-class="{
+    collapsed: nav.collapsed && !isMobile,
+    'hide-mobile-nav': !nav.showMobileNav && isMobile,
+    'hover-secondary-nav-pf': sidebar.secondaryOpen && !isMobile,
+    'show-mobile-nav': nav.showMobileNav && isMobile,
+    'show-mobile-secondary': nav.showMobileNav && sidebar.showMobileSecondary && isMobile
+  }"
+  on-esc="closeNav()">
+  <nav class="nav-vertical-primary">
+    <ul class="list-group">
       <li
         ng-repeat="primaryItem in navItems"
-        ng-class="{ active: primaryItem === activePrimary }"
-        ng-if="show(primaryItem)">
-        <a ng-if="primaryItem.href" ng-href="{{navURL(primaryItem.href)}}">
-          <span class="{{primaryItem.iconClass}}"></span> {{primaryItem.label}}
+        ng-class="{
+          active: primaryItem === activePrimary,
+          'is-hover': primaryItem.isHover,
+          'secondary-nav-item-pf': primaryItem.secondaryNavSections.length
+        }"
+        ng-if="show(primaryItem)"
+        ng-mouseenter="onMouseEnter(primaryItem)"
+        ng-mouseleave="onMouseLeave(primaryItem)"
+        class="list-group-item">
+        <a ng-if="primaryItem.href" ng-href="{{navURL(primaryItem.href)}}" ng-click="itemClicked(primaryItem)">
+          <span class="{{primaryItem.iconClass}}"></span> <span class="list-group-item-value">{{primaryItem.label}}</span>
         </a>
-        <a ng-if="!primaryItem.href" href="" data-toggle="dropdown" class="dropdown-toggle">
-          <span class="{{primaryItem.iconClass}}"></span> {{primaryItem.label}} <span class="fa fa-angle-right"></span>
+        <a
+          ng-if="!primaryItem.href"
+          href=""
+          ng-click="itemClicked(primaryItem)">
+          <span class="{{primaryItem.iconClass}}"></span> <span class="list-group-item-value">{{primaryItem.label}}</span>
         </a>
+
+        <!-- mobile secondary nav -->
```

Comment:
I don't think this is just for mobile? It's secondary nav for desktop, too, right?

If so, let's update the comment.